content/spaces/setup.textile

```diff
@@ -0,0 +1,77 @@
+---
+title: SDK setup
+meta_description: "Install and instantiate the Spaces SDK."
+languages:
+  - javascript
+---
+
+Use these instructions to setup the Spaces SDK.
+
+h2(#api). Get an API key
```

Comment:
```suggestion
h2(#api). Authentication
```

Reply:
I think this should be a brief section about authentication, the two types available, links to core docs and permissions. Just like you've done but with some tweaks to the structure and positioning.